### ISSUE TITLE
orb bug

### ISSUE THREAD

**Issue by Sleuth56** [ObsidianBlk/Project-AL#2]
I think I found a bug.  Upon hitting this orb you contenue to move backwards until hitting the wall or until you push a button
![screenshot from 2018-04-24 19 23 37](https://user-images.githubusercontent.com/34520077/39218903-2eed9dec-47f5-11e8-96b6-b70b1e1a5312.png)

P.S.  
This maby a bug I'm not sure
![screenshot from 2018-04-24 19 16 09](https://user-images.githubusercontent.com/34520077/39218712-01d5673c-47f4-11e8-84b5-2424f95e179c.png)

**Comment by ObsidianBlk** [comment]
You're absolutely right about the orb. To be specific, I think it has to do with the ramp, actually. Will have to check. As for the second one... no, that's not a bug. That was a "secret" :)
